features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/source/thread_tasklet.c

```diff
@@ -228,67 +229,55 @@ void thread_tasklet_configure_and_connect_to_network(void)
         NET_6LOWPAN_THREAD);
 
     // Link configuration
-    memcpy(thread_tasklet_data_ptr->link_config.name, "Arm Powered Core", 16);
-
     thread_tasklet_data_ptr->link_config.panId = MBED_MESH_API_THREAD_CONFIG_PANID;
     TRACE_DETAIL("PANID %x", thread_tasklet_data_ptr->link_config.panId);
 
```

Comment:
Because you don't know if channel 27 is valid or not. Don't assume IEEE 802.15.4 2.4 Ghz channels...